app/models/event.rb

```diff
@@ -28,18 +32,32 @@ class Event < ActiveRecord::Base
   # Helpers ####################################################################
 
   def attended_users
-    self.attendances.where(present: true).collect(&:user)
+    self.attendances.where(present: true).collect(&:user).sort_by {
+      |u| [u.last_name, u.first_name]
+    }
   end
 
   def absent_users
-    self.attendances.where(present: false).collect(&:user)
+    self.attendances.where(present: false).collect(&:user).sort_by {
```

Comment:
Avoid using {...} for multi-line blocks.<br>Prefer `map` over `collect`.<br>Redundant `self` detected.